Makefile

```diff
@@ -1,4 +1,4 @@
-SERVICE ?= quay.io/oamizur/introspector:latest
+SERVICE := $(or ${SERVICE},${SERVICE},quay.io/oamizur/introspector:latest)
```

Comment:
Why twice ${SERVICE}

Reply:
You need to remove one ${SERVICE}

Reply:
the first is a condition, the second is if's the condition is true and the third part is the else

Reply:
i think.. but it works

Reply:
you are right it works with two as well